.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -17,32 +18,26 @@ Questions for PR author:
 -->
 
 <!--
-Images or gifs: Include before and after screenshots or gifs/videos when it makes sense.
+Images or gifs: Include before and after screenshots or gifs/videos as appropriate.
 -->
 
-#### Testing instructions
+## How to test this PR
 
 <!--
 Testing instructions: How should this be tested and how can a reviewer test the end-user functionality? Are there known issues that you plan to address in a future PR? Are there any side effects that readers should be aware of?
+Use the testing instructions from the linked issue as a starting point.
 -->
 
 <!--
 Add as many details as possible to help others reproduce the issue and test the fix.
 "Before / After" screenshots can also be very helpful when the change is visual.
 -->
 
-*
-
--------------------
+1. Go to …
+2. Click on …
 
-- [ ] Added changelog entry (or does not apply)
-- [ ] Covered with tests (or have a good reason not to test in description ☝️)
-- [ ] Tested on mobile (or does not apply)
-
-**Post merge**
-
-<!--
-Make sure you edit the page for the current release when adding testing instructions.
-We often create a blank page ahead of time for the next release.
--->
+## Product impact
+<!-- What products will this PR ship in? -->
 
+- [ ] Will this PR affect WooCommerce Subscriptions? yes/no/tbc, add issue ref
+- [ ] Will this PR affect WooCommerce Payments? yes/no/tbc, add issue ref
```

Comment:
```suggestion
- [ ] Added changelog entry (or does not apply)
- [ ] Will this PR affect WooCommerce Subscriptions? yes/no/tbc, add issue ref
- [ ] Will this PR affect WooCommerce Payments? yes/no/tbc, add issue ref
```

Reply:
Good thinking!

Though I figure the changelog entries will get added to the product repos directly. We need to nail down the changelog process.